Automattic/themes
# Update the Professional Business theme to support FSE header

#### Changes proposed in this Pull Request:

This change will update the professional business theme to pull in the header styles into the editor through the style-editor.scss file. This means that when using the full site editing plugin, you will see almost a one to one representation of the header design in the editor vs the front end:

<img width="1416" alt="Screen Shot 2019-07-11 at 10 38 51 PM" src="https://user-images.githubusercontent.com/1464705/61104870-b002ca80-a42c-11e9-852b-86f3ec9f12ee.png">

To test this:

* Install the professional business theme locally
* Install the latest master branch build of the full site editing plugin from Calypso
* Create a new page
* Edit the header of the page and delete all content
* Paste the following into the editor: https://codepen.io/apeatling/pen/VJNZVJ and update/publish the header template.
* Confirm that you see a two column header that matches the screenshot above, and this looks identical on the front end of the site.
* Confirm that with the full site editing plugin disabled, the old template file based header works as expected.

We are yet to be able to support multiple header templates, which is why you need to paste the header block template code at this point. More about that here: https://github.com/Automattic/wp-calypso/issues/34610

#### Related issue(s):

* https://github.com/Automattic/wp-calypso/issues/34387 
* https://github.com/Automattic/wp-calypso/pull/34568
